client/src/Components/Employees/EmployeeList.js

```diff
@@ -4,49 +4,16 @@ import { FlexColumn, Divider, FlexRow } from "custom-components";
 import EmployeeListItem from "./EmployeeListItem";
 import { Link } from "react-router-dom";
 
+import taskPropertyAssign from './taskPropertyHelper';
+
 const EmployeeList = props => {
   const {
-    employees,
-    tasks,
-    properties,
     numPages,
     page,
     handlePageChange
   } = props;
 
-  if (tasks && properties) {
-    const currentTime = Date.now();
-    const newTasks = [];
-    tasks.forEach(item => {
-      const newTaskData = {};
-      if (Date.parse(item.endDate) < currentTime) {
-        newTaskData.overdue = true;
-      } else if (Date.parse(item.startDate) < currentTime) {
-        newTaskData.todayTask = true;
-      }
-      newTaskData.employee = item.assignedTo._id;
-      newTasks.push(newTaskData);
-    });
-    employees.forEach(employee => {
-      employee.overdue = 0;
-      employee.todayTask = 0;
-      employee.properties = 0;
-      newTasks.forEach(task => {
-        if (task.employee === employee._id) {
-          if (task.overdue) {
-            employee.overdue++;
-          } else if (task.todayTask) {
-            employee.todayTask++;
-          }
-        }
-      });
-      properties.forEach(property => {
-        if (property.assistants.includes(employee._id)) {
-          employee.properties++;
-        }
-      });
-    });
-  }
+  const modEmployees = taskPropertyAssign(props);
```

Comment:
Would rename modifiedEmployees and/or add a comment above this just to explain what it's doing and why you're using this variable since it's not clear just from reading through.

Reply:
Good point. Adding now.